package.json

```diff
@@ -7,8 +7,6 @@
       "./packages/*"
     ],
     "nohoist": [
-      "**/react",
```

Comment:
@vr2amesh can you undo these changes to package.json to see if the build completes successfully?

Reply:
@vr2amesh I'm seeing this error when running `yarn run build` locally in this repo: 
`| ERROR in ../../node_modules/@toruslabs/base-controllers/node_modules/@ethereumjs/util/dist/ssz.js 4:14-39
 | Module not found: Error: Can't resolve '@chainsafe/ssz' in '/Users/katleight/Blocknative/web3-onboard-capsule/node_modules/@toruslabs/base-controllers/node_modules/@ethereumjs/util/dist'`

Reply:
@leightkt I think there are some issues with the merge I performed. I tried this on a fresh branch and things seem to be in order. Let me recreate the branch shortly and then will ping you right away here

Reply:
@leightkt Updated the yarn.lock file to match a proper fresh branch. Looks like we've got the build issues fixed

Reply:
looking good! thanks for all your hard work on this!

Reply:
@leightkt Thank you for the quickness in response. It is sincerely appreciated :)